modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-additional-permissions-for-cluster-config_{context}"]
+= Providing additional permissions for cluster configuration
+
+For cluster-configuration management you can add permissions to an Argo CD instance. 
```

Comment:
The current phrasing makes "Providing additional permissions for cluster configuration" sound like it might be an optional task. If the task is required, reword it accordingly.

Reply:
@rolfedh the first suggestion goes perfectly including the same. "You can add permissions to an Argo CD instance to support cluster configuration."

Reply:
@gabriel-rh changed the word "support" to "manage".